ng-dialog/ng-dialog.d.ts

```diff
@@ -83,6 +136,12 @@ declare module angular.dialog {
 		closeByDocument?: boolean;
 
 		/**
+		 * default : false
+		 */
+		closeByNavigation?: boolean;
```

Comment:
Ah but it is within code, https://github.com/likeastore/ngDialog/blob/master/js/ngDialog.js#L48 .. not sure what it does, but if you know you can add some comments to the code :)